src/tox/session/__init__.py

```diff
@@ -220,6 +220,26 @@ def subcommand_test(self):
             retcode = self._summary()
         return retcode
 
+    def _add_parallel_summaries(self):
+        import json
+        if 'testenvs' not in self.resultlog.dict:
+            return
+        for venv in self.venv_dict.values():
+            testenvs = self.resultlog.dict['testenvs']
+            if venv.name not in testenvs:
+                continue
+            result_json_path = venv.path.join('.tox-result.json')
```

Comment:
you're not indexing the sub-calls here, how should this load the correct value?

Reply:
the venv.path is already unique for each environment, why should I use the index?

Reply:
this really should be a constant (inside tox.constants) re-used between files rather than hard coded like this